addons/website_sale/views/templates.xml

```diff
@@ -1027,11 +1027,11 @@
                         <div class="col-lg-6 d-flex align-items-center">
                             <ol class="breadcrumb p-0 mb-2 m-lg-0">
                                 <li class="breadcrumb-item o_not_editable">
-                                    <a t-att-href="keep(category=0)">All Products</a>
+                                    <a class="keep_filters" t-att-href="keep(category=0)">All Products</a>
```

Comment:
If the sole purpose of this class is to be a selector, please use a name instead :)